service/user.ts

```diff
@@ -88,8 +88,11 @@ export async function getCurrentUser(): Promise<User.AsObject> {
  * @param {string} user
  * @returns {Promise<User.AsObject>}
  */
-export async function getUser(user: string): Promise<User.AsObject> {
-  return http.get(`users/${user}/`);
+export async function getUser(user: string): Promise<User> {
```

Comment:
Looks like this is still using `User` from `proto/api_pb` or at least the docstring is

Reply:
true, I forgot to update the docstring. It is using the new backend though.